src/routes/organizations/organizations.controller.ts

```diff
@@ -69,6 +70,26 @@ export class OrganizationsController {
     });
   }
 
+  @Post('/create-with-user')
+  @ApiOkResponse({
+    description: 'Organizations created',
+    type: CreateOrganizationResponse,
```

Comment:
@usame-algan, would you benefit from having the new user ID in the response?

Reply:
Don't think we use the user id in any of the requests so it would be fine not returning it.